bin/smart-proxy-openscap-send

```diff
@@ -37,8 +37,8 @@ end
 
 begin
   Proxy::OpenSCAP::send_spool_to_foreman
-rescue StandardError => e
-  logger.debug e.backtrace.join("\n\t")
-  logger.error "Failed to send SCAP results to the Foreman server: #{e}"
+rescue RuntimeError => e
```

Comment:
why is this changed to RuntimeError? I see that StandardError is being catched in spool_forwarder.rb now but if it's risen somewhere else we should still catch them (e.g. IOError).